subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,7 @@ import static org.hamcrest.MatcherAssert.assertThat
 
 abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
 
-    protected static final THIRD_PARTY_LIB_COUNT = 145
+    protected static final THIRD_PARTY_LIB_COUNT = 167
 
```

Comment:
```
➜  gradle git:(rchaves/maven-dependencies-20596) ✗ find ~/gradle-installs/pr-20596 -name "*.jar" | wc -l
     255
➜  gradle git:(rchaves/maven-dependencies-20596) ✗ find ~/gradle-installs/master -name "*.jar" | wc -l
     233
     ```

Reply:
@bot-gradle test this

Reply:
do you know what these extra libraries are?  I thought this only removed a few (not >20) libraries when we removed them in 7.0?

Reply:
```
diff /tmp/master-libs.txt /tmp/pr-21202-libs.txt              
130a131
> lib/plugins/aopalliance-1.0.jar
139a141
> lib/plugins/cdi-api-1.0.jar
140a143
> lib/plugins/commons-lang3-3.12.0.jar
195a199
> lib/plugins/guice-4.2.3-no_aop.jar
213a218
> lib/plugins/jsr250-api-1.0.jar
217a223
> lib/plugins/maven-artifact-3.6.3.jar
218a225,226
> lib/plugins/maven-compat-3.6.3.jar
> lib/plugins/maven-core-3.6.3.jar
219a228,229
> lib/plugins/maven-model-builder-3.6.3.jar
> lib/plugins/maven-plugin-api-3.6.3.jar
220a231,237
> lib/plugins/maven-resolver-api-1.4.1.jar
> lib/plugins/maven-resolver-connector-basic-1.4.1.jar
> lib/plugins/maven-resolver-impl-1.4.1.jar
> lib/plugins/maven-resolver-provider-3.6.3.jar
> lib/plugins/maven-resolver-spi-1.4.1.jar
> lib/plugins/maven-resolver-transport-http-1.4.1.jar
> lib/plugins/maven-resolver-util-1.4.1.jar
222a240
> lib/plugins/maven-shared-utils-3.2.1.jar
224a243,244
> lib/plugins/org.eclipse.sisu.inject-0.3.4.jar
> lib/plugins/org.eclipse.sisu.plexus-0.3.4.jar
225a246,247
> lib/plugins/plexus-classworlds-2.6.0.jar
> lib/plugins/plexus-component-annotations-2.1.0.jar
230a253
> lib/plugins/wagon-provider-api-3.3.4.jar
```

Reply:
@big-guy It is basically most-of-Maven vs maven-settings-builder.